app/Main.hs

```diff
@@ -41,37 +37,32 @@ import Network.Wai.Middleware.RequestLogger (logStdoutDev)
 -- warp
 import Network.Wai.Handler.Warp (run)
 
+import qualified Tagger.Database as DB
```

Comment:
imports from inside the project as listed above in alphabetical order. Would be nice to keep them together

Reply:
This will be addressed with an automatic formatter, so I suggest ignoring it for now.